api/src/apps/ovfiets/importer.py

```diff
@@ -0,0 +1,145 @@
+import logging
+
+from django.contrib.gis.geos import Point
+from django.db.models import FieldDoesNotExist
+from django.utils import timezone
+
+from apps.base.models import District, Neighbourhood
+from apps.ovfiets.models import OvFiets, OvFietsRaw
+
+logging.basicConfig(level=logging.INFO, format='%(message)s')
+log = logging.getLogger(__name__)
+log.setLevel(logging.INFO)
+
+
+class OvFietsImporter:
+    '''
+    Class used to unpack OvFietsRaw records, clean them then
+    insert them into OvFiets model
+
+    altered_keys is a dictionary containing the keys in the raw instances
+    that need to be altered to the model field
+    '''
+    altered_keys = {
+        'openingHours': 'opening_hours',
+        'locationCode': 'location_code',
+        'stationCode': 'station_code',
+        'fetchTime': 'fetch_time',
+        'rentalBikes': 'rental_bikes'
+    }
+
+    def clean_key(self, key):
+        '''
+        Checks if key needs to be altered
+        '''
+        if key in self.altered_keys:
+            key = self.altered_keys[key]
+        return key
+
+    def set_to_model_field(self, model_instance, key, value):
+        '''
+        Checks if field exists then saves it to the model instance
+
+        Specific to this datasource, we dump all non existing fields to
+        the jsonfield `unmapped`
+        '''
+        try:
+            field = model_instance._meta.get_field(key)
+            setattr(model_instance, field.name, value)
+        except FieldDoesNotExist:
+            log.debug(f"Field {key} does not exist. Adding to unmapped")
+            model_instance.unmapped.update({key: value})
+
+    def clean_value(self, key, value):
+        '''
+        Parses the fetch_time timestamp and makes aware (timezone)
```

Comment:
"Clean the value so it can be stored in the model. 
Currently only modifies the fetch_time key to make it timezone aware."